src/XamlStyler.Extension.Windows.VS2019/XamlStyler.Package.vsct

```diff
@@ -235,9 +222,6 @@
       <IDSymbol name="GroupIDXamlStylerContextCrossProjectMultiSolutionFolderGroup" value="0x105a" />
       <IDSymbol name="GroupIDXamlStylerContextCrossProjectMultiProjectFolderGroup" value="0x105b" />
 
-      <!-- This is the command set for the document tab context menu -->
-      <IDSymbol name="GroupIDXamlStylerContextDocumentTabGroup" value="0x1070" />
```

Comment:
This PR removes format option from file tab context menu:

<img width="242" alt="image" src="https://github.com/Xavalon/XamlStyler/assets/1414423/0232988b-fb6c-4c4c-aa07-a5ff2e73c417">

I suspect it might be this one but did not test myself.

Reply:
Sorry, missed this comment previously.
The current (prior to this PR) version adds the "Format XAML" option to the context menu on the tab of any document window. Am I right to assume this should be filtered to just xaml files also?

Reply:
Yes, this is ideally filtered to xaml (and axaml) files.